flink-core/src/main/java/org/apache/flink/api/common/functions/RuntimeContext.java

```diff
@@ -56,6 +58,13 @@
 @Public
 public interface RuntimeContext {
 
+    /**
+     * The ID of the current job. Empty if the execution happens outside of any job context (e.g.
+     * standalone collection executor). The returned ID should NOT be used for any job management
```

Comment:
I'd add that JobID may change upon (manual?) restart.

Reply:
Done.